test/reorderError2.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>QUnit Main Test Suite</title>
+	<link rel="stylesheet" href="../dist/qunit.css">
+	<script src="../dist/qunit.js"></script>
+	<script>
+	// Second Case.
```

Comment:
We may repeat the comments [suggested above](https://github.com/jquery/qunit/pull/907/files#r48490014)